qemu/tests/drive_mirror.py

```diff
@@ -62,7 +62,11 @@ def get_target_image(self):
 
     def get_device(self):
         params = super(DriveMirror, self).parser_test_args()
-        image_file = storage.get_image_filename(params, self.data_dir)
+        image_name = params["image_name"]
+        image_format = params["image_format"]
+        image_name += ".%s" % image_format
+        image_file = utils_misc.get_path(self.data_dir, image_name)
+        logging.info("image filename: %s" % image_file)
```

Comment:
a single string formatting will do the work as we discussed before.
Please remove the period in the commit head.